Sombra.Web/Infrastructure/Authentication/AuthorizationAttributes/DefaultAttribute.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Microsoft.AspNetCore.Authorization;
+
+namespace Sombra.Web.Infrastructure.Authentication.AuthorizationAttributes
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+    public class DefaultAttribute : AuthorizeAttribute
```

Comment:
Slechte naamgeving. Je attribute komt er uit te zien als [Default], wat niet heel veelzeggend is